WhaleSpotting/react-app/src/styles/Card.scss

```diff
@@ -23,6 +23,18 @@
     }
 }
 
+.buttons-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    flex: 1 1;
+    align-items: center;
+}
+
+.hidden {
+    display: none;
+}
```

Comment:
If we do the hidden comment in card we don't need this

Reply:
:)